api/management/swagger-api_management.yaml

```diff
@@ -1827,6 +1858,25 @@ paths:
       responses:
         200:
           description: successful operation  
+  /realms/{realm}/identity-provider/instances:
+    get:
+      tags:
+      - Brokering
+      summary: Get the list of all IDPs in the realm
+      parameters:
+      - name: realm
+        in: path
+        description: realm name (not id!)
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: Login provider id
+          content:
+            application/json:
```

Comment:
Detail, you could define the response as an array of IdentityProvider.

Reply:
Done